packages/app/lib/utils/utils.ts

```diff
@@ -66,9 +66,11 @@ export const hasData = ({ event }: { event: IExtendedEvent }) => {
 }
 
 export const getImageUrl = (image: string) => {
-  //return `https://raw.githubusercontent.com/streamethorg/streameth-platform/main/images${image}`
-  // temp revert to github
-  return `https://streameth-production.ams3.digitaloceanspaces.com${image}`
+  const spaceStorage = process.env.SPACE_STORAGE_URL || null
+  if (!spaceStorage) {
+    throw new Error('No SPACE STORAGE URL key found')
+  }
+  return process.env.NEXT_PUBLIC_SPACE_STORAGE + '/' + image
```

Comment:
Kindly review the usage in ImageUpload component, you added a leading `\` before image, this will cause a double `\\` in the resulting link 
`const [preview, setPreview] = useState(
    value ? getImageUrl('/' + path + '/' + value) : ''
  )`